### DISCUSSION TITLE
Why boundaries is std::list

### DISCUSSION THREAD

**Discussion by ondra-novak** [open-telemetry/opentelemetry-cpp#1863] [Q&A] [answered]
https://github.com/open-telemetry/opentelemetry-cpp/blob/b8b715faf9a364767bb851de763ccef983d65ccf/sdk/include/opentelemetry/sdk/metrics/data/point_data.h#L59

Just explain me, why list is used here. Unless you need to insert into or erase from inside of the list, this container is very slow and you are loosing a lot of performance here.

This has been changed from std::vector in commit a7e814a632fdaee0c76246358ffad811ac06a7e9 without any sign of need such a change and without explanation.

**Comment by lalitb** [comment, marked as answer]
I can't recall why this was done now, but agree std::vector ( or perhaps std::array) would be more efficient. Creating a issue out of it. Thanks for raising the query.
